website/src/settings.mjs

```diff
@@ -29,7 +30,15 @@ export const defaultSettings = {
   audioDeviceName: defaultAudioDeviceName,
 };
 
-export const settingsMap = persistentMap('strudel-settings', defaultSettings);
+let search = null;
+if (typeof window !== 'undefined') {
+  search = new URLSearchParams(window.location.search);
+}
+// if running multiple instance in one window, it will use the settings for that instance. else default to normal
+const instance = parseInt(search?.get('instance') ?? '0');
+const settings_key = `strudel-settings${instance > 0 ? instance : ''}`;
+
+export const settingsMap = persistentMap(settings_key, defaultSettings);
```

Comment:
could be cool the clone the settings of the main repl or latest udel repl instead of using defaultSettings? tho it's only relevant on first use

Reply:
Yes that makes sense